packages/manager/apps/vrack-services/src/public/translations/vrack-services/subnets/Messages_fr_FR.json

```diff
@@ -7,12 +7,11 @@
   "subnetCreationError": "Erreur lors de la création du sous-réseau : {{error}}",
   "subnetNameLabel": "Nom du sous-réseau (facultatif)",
   "subnetNamePlaceholder": "Mon sous-réseau",
-  "defaultSubnetName": "SUB-XXXXX",
   "cidrLabel": "Plage d'adresses du sous-réseau (CIDR)",
   "cidrPlaceholder": "10.0.0.0/24",
   "defaultCidr": "10.0.0.0/24",
   "serviceRangeLabel": "Plage d'adresses de service (CIDR)",
-  "serviceRangePlaceholder": "Sélectionnez une plage d'adresses",
+  "serviceRangePlaceholder": "10.0.0.0/29",
```

Comment:
Sonar doesn't like hardcoded ip addresses : it creates an issue

And is it really better to keep them in the code ? We don't do anything with them in the frontend code and the PU will have easier time changing the values if they decide so.

Reply:
We can't just update all the translation files ?

And if we want a different default value for each region, it could be beneficial though (but it will happen either)

Reply:
@chipp972, no we are not supposed to modify anything in locale files apart from `Messages_fr_FR.json`

Reply:
Ok but it's still the placeholder of the input, we don't compute anything technical with this value in the front beside passing it to the api.